content/ctl_manage.md

```diff
@@ -27,21 +27,6 @@ The Chef management console includes a command-line utility named
 cleanse (reset the Chef management console to initial configuration
 settings), and uninstall the Chef management console.
 
-## cleanse
```

Comment:
This entire page is about the management console

Reply:
It is in `Legacy`. As far as I can tell, the `cleanse` command no longer exists in chef-manage-ctl.

Reply:
I could be wrong, though.